source/guide_castopod.rst

```diff
@@ -0,0 +1,193 @@
+.. author:: André Jaenisch <https://jaenis.ch>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: fediverse
+.. tag:: ActivityPub
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/castopod.svg
+      :align: center
+
+########
+Castopod
+########
+
+.. tag_list::
+
+Castopod_ is a free and open-source podcast hosting platform.
+
+It forms one part of the Fediverse, an interconnected and
+decentralized network of independently operated servers.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>` and its package manager :manual_anchor:`composer <lang-php#package-manager>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`Cron <daemons-cron>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+Castopod_ is released under the `GNU Affero General Public License v3.0`_.
+All relevant information can be found in the LICENSE.md_ file in the
+repository of the project.
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the version 8.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use php 8.1
+ Selected PHP version 8.1
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Setup a new MySQL database for Castopod:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_castopod"
+  [isabell@stardust ~]$
+
+Your castopod URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+.. note::
+  Castopod should be installed in a domain of its own (or in a subdomain),
+  not in a subdirectory of a domain.
+
+Installation
+============
+
+First, go one level above your :manual:`DocumentRoot <web-documentroot>`
+and download the latest version. Make sure, it is a Castopod Package and not
+the source code. At time of this writing this is v1.0.0-beta.14.
+
+TODO: the permalink to latest version requires login:
```

Comment:
that is bad, is it possible to get a link where you just need to lookup the latest version and replace it within the link?